hack/lib/tilt/Tiltfile

```diff
@@ -95,6 +95,7 @@ k8s_yaml(local([
     '--data-value', 'namespace=supervisor',
     '--data-value', 'image_repo=image/supervisor',
     '--data-value', 'image_tag=tilt-dev',
+    '--data-value', 'log_level=debug',
```

Comment:
How would we change this during development (same question applies to a few places)?

Reply:
I _think_ the best way to change this during development is to go into the `Tiltfile` and update `log_level=debug` to `log_level=trace` (or whatever you want). `tilt` _should_ pick this change up and deploy everything again. Personally, I have been running into an issue with running `tilt` locally such that it does _not_ deploy everything again, but I think that is user error/a `tilt` bug. The eject button here is to Ctrl-C the `tilt` process and run it again, I believe.